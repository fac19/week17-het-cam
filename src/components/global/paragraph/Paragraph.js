import styled from "styled-components";

const Paragraph = styled.p`
  text-align: center;
  line-height 1.2;
`;

export default Paragraph;
