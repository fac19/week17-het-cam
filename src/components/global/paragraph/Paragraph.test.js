import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Paragraph from "./Paragraph";

test("Paragraph component is styled", () => {
  const tree = renderer.create(<Paragraph />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("text-align", "center");
});
