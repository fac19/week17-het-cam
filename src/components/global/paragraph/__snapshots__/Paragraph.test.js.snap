// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph component is styled 1`] = `
.c0 {
  text-align: center;
  line-height: 1.2;
}

<p
  className="c0"
/>
`;
